Description: Enable building with Qt 5.14
Origin: upstream, https://commits.kde.org/apper/1815d19f01861827bac9a862d77d098215a39ab9
Forwarded: yes
Applied-Upstream: commit:1815d19f01861827bac9a862d77d098215a39ab9
Reviewed-By: John Scott <jscott@posteo.net>
Last-Update: 2020-07-28
---

--- apper-1.0.0.orig/Apper/ApperKCM.cpp
+++ apper-1.0.0/Apper/ApperKCM.cpp
@@ -35,6 +35,7 @@
 #include <QSignalMapper>
 #include <QTimer>
 #include <QPointer>
+#include <QKeyEvent>

 #include <PackageModel.h>
 #include <ApplicationSortFilterModel.h>
